File: src/main/kotlin/com/cleitech/receipt/shoeboxed/domain/Documents.java
package com.cleitech.receipt.shoeboxed.domain;

import java.util.LinkedList;
import java.util.List;

/**
 * @author Pierrick Puimean-Chieze on 23-04-16.
 */
public class Documents {

    private LinkedList<Document> documents = new LinkedList<>();

    public LinkedList<Document> getDocuments() {
        return documents;
    }

    public void setDocuments(List<Document> documents) {
        this.documents.clear();
        this.documents.addAll(documents);
    }
}

File: src/main/kotlin/com/cleitech/receipt/shoeboxed/domain/Documents.kt
package com.cleitech.receipt.shoeboxed.domain

import java.util.*

/**
 * @author Pierrick Puimean-Chieze on 23-04-16.
 */
class Documents {

    val documents = LinkedList<Document>()

    fun setDocuments(documents: List<Document>) {
        this.documents.clear()
        this.documents.addAll(documents)
    }
}
